aea/test_tools/docker_image.py

```diff
@@ -25,17 +25,13 @@
 import subprocess  # nosec
 import time
 from abc import ABC, abstractmethod
-from typing import Any, Generator
+from typing import Generator, cast
 
+import docker
 import pytest
-
-
-try:
-    from docker import DockerClient
-    from docker.models.containers import Container
-except ImportError:
-    DockerClient = Any
-    Container = Any
```

Comment:
Please revert. This is on purpose to avoid having  to make docker a framework dependency